apollo-ios/Sources/ApolloWebSocket/WebSocketTransport.swift

```diff
@@ -207,11 +218,6 @@ public class WebSocketTransport {
                                                 kind: .neitherErrorNorPayloadReceived)
             responseHandler(.failure(websocketError))
           }
-        } else {
```

Comment:
An alternative approach to silencing this error path is to move unsubscribing IDs into another array that can be checked against if the ID is not found in `subscribers`. This would ensure we're discarding a message for an unsubscribing handler but I'm not sold on that being necessary.

_The unsubscribing array should be of `[String]` type since there would be no need to store the handling closure again._